ote/src/cljs/ote/views/pre_notices/pre_notice.cljs

```diff
@@ -0,0 +1,213 @@
+(ns ote.views.pre-notices.pre-notice
+  "Pre notice main form"
+  (:require [reagent.core :as r]
+            [cljs-react-material-ui.reagent :as ui]
+            [ote.localization :refer [tr tr-key]]
+            [ote.app.controller.pre-notices :as pre-notice]
+            [ote.ui.buttons :as buttons]
+            [ote.ui.form-fields :as form-fields]
+    ;; db
+            [ote.db.transport-operator :as t-operator]
+            [ote.db.common :as db-common]
+            [ote.db.transit :as transit]
+            [ote.ui.form :as form]
+            [ote.ui.common :as common]
+            [cljs-react-material-ui.icons :as ic]
+            [ote.style.form :as style-form]
+            [stylefy.core :as stylefy]
+            [ote.ui.leaflet :as leaflet]))
+
+(def notice-types [:termination :new :schedule-change :route-change :other])
+
+(defn select-operator [e! operator operators]
+  [:div
+   [:div.row
+    [form-fields/field
+     {:label       (tr [:field-labels :select-transport-operator])
+      :name        :select-transport-operator
+      :type        :selection
+      :show-option ::t-operator/name
+      :update!     #(e! (pre-notice/->SelectOperatorForNotice %))
+      :options     operators
+      :auto-width? true}
+     operator]]
+   [:div.row.col-xs-12.col-sm-12.col-md-8
```

Comment:
why not use `ote.ui.common/table2` to show info, do we need form fields for read-only data... we could just show the information without having editable components that are disabled

Reply:
Jussi oli piirtänyt nuo disabloidut kentät tuonne. Minusta tuon ois voinut tehdä nätimminkin.